css/style.css

```diff
@@ -1,5 +1,449 @@
-*{
+@font-face {
+  font-family: "GT Walsheim";
+  src: local("GT Walsheim Regular"), local("GTWalsheim-Regular"),
+    url("../fonts/GTWalsheim-regular.woff2") format("woff2");
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "GT Walsheim";
+  src: local("GT Walsheim Medium"), local("GTWalsheim-Medium"),
+    url("../fonts/GTWalsheim-medium.woff2") format("woff2");
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "GT Walsheim";
+  src: local("GT Walsheim Bold"), local("GTWalsheimBold"),
+    url("../fonts/GTWalsheimBold.woff2") format("woff2");
+  font-weight: bold;
+  font-style: normal;
+  font-display: swap;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "GT Walsheim";
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  width: 95%;
+  max-width: 1200px;
+  margin: auto;
+  overflow: hidden;
+}
+
+.logo {
+  margin-right: 16rem;
+}
+
+.button-small {
+  font-size: 16px;
+  line-height: 18.75px;
+  font-weight: bold;
+  letter-spacing: -0.11px;
+  background: #ffffff;
+  border-radius: 20px;
+  padding: 7px 30px;
+  color: #00343d;
+  text-decoration: none;
+  margin-left: 30px;
+}
+
+.btn {
+  width: 251px;
+  text-align: center;
+  display: inline-block;
+  padding: 18px 22px;
+  font-size: 18px;
+  line-height: 21px;
+  letter-spacing: 0.594px;
+  text-transform: uppercase;
+  border-radius: 35px;
+  font-weight: bold;
+  color: #00343d;
+  background-color: #ffffff;
+  border: none;
+  cursor: pointer;
+  text-decoration: none;
+  transition: 0.25s;
+  margin-top: 40.75px;
+  box-shadow: 0px 1px 2px 0px rgba(0, 52, 61, 0.44);
+}
+
+.btn-green {
+  color: #00343d;
+  background: #0bd8a2;
+}
+.btn-green:hover {
+  color: #ffffff;
+}
+
```

Comment:
This class should be a modifier class `.button--green` and `.button--green:hover`
Please apply BEM @Jefer-Max